src/content/reference/react/index.md

```diff
@@ -110,6 +110,7 @@ To prioritize rendering, use one of these Hooks:
 
 These Hooks are mostly useful to library authors and aren't commonly used in the application code.
 
+- [`use`](/reference/react/use) lets you read the value of a resource like a [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) or [context](/learn/passing-data-deeply-with-context)
```

Comment:
imo `use` is or will be a lot more important than useDebugValue, useId, useSyncExternalStore. if you want to leave it here while it's only in canary that's probably OK but before stable it should probably move up, maybe to its own section?

Reply:
Good callout, I added a "Resource Hooks" section.